package.json

```diff
@@ -31,6 +32,7 @@
     "lz-string": "^1.4.4",
     "marray": "^1.1.6",
     "quasar": "^1.15.19",
+    "tiny-async-pool": "^1.2.0",
```

Comment:
I don't think these are required in the main package.json. The functions folder has its own because it runs on the server, and uses a specific version of node.

Reply:
I added them before I knew to run `npm install` in the functions folder. I'll clean it and try again to confirm that takes care of it.

Reply:
Fixed.